client/my-sites/subscribers/components/subscriber-totals/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { translate } from 'i18n-calypso';
+import { SubscribersFilterBy } from '../../constants';
+import './style.scss';
+
+type SubscriberTotalsProps = {
+	totalSubscribers: number;
+	filteredCount: number;
+	filterOption: SubscribersFilterBy;
+	searchTerm: string;
+	isLoading: boolean;
+};
+
+const getFilterLabel = ( filterOption: SubscribersFilterBy, count: number ): string => {
+	switch ( filterOption ) {
+		case SubscribersFilterBy.Paid:
+			return count === 1 ? translate( 'paid subscriber' ) : translate( 'paid subscribers' );
+		case SubscribersFilterBy.Free:
+			return count === 1 ? translate( 'free subscriber' ) : translate( 'free subscribers' );
```

Comment:
I think best to use `translate` singular/plural form, which uses a count prop

Reply:
Ah I think the reason you went with this form was because we don't have a variable in the string  right? Or was it just overight? (Cause I see we do use the plural call below)

Reply:
I was also getting a type error, but I think I can fix it with casting. I'll open a PR to use the count prop throughout the file.